sdk/template/azure-template/setup.py

```diff
@@ -43,8 +43,6 @@
     classifiers=[
         "Development Status :: 4 - Beta",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
```

Comment:
Maybe we also want to add 3.10?

Reply:
Also add
Programming Language :: Python :: 3 :: Only